### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by devowhippit**
* Add Security Plugins to composer.json
* Add Composer scripts
* Add phpcs.xml standard
* Consolidate .cfg files into one bin.cfg file
* Update documenation for all items, include a table of contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,25 +69,25 @@ to start them. You can use the *-d* flag (`docker-compose up -d`) to run in deta
...
 
-If you install WordPress in a directory other than **/wp** you will need to change the configuration in **wp.cfg**.
+Config Section    | Description
+------------------|-
+Colors            | These are the colors used for Slack and other message highlighting. They currently are set to match the NYC Opportunity brand.
+Domain            | The production domain, CDN, and path for distributed *.js* files go here.
+GitHub            | `GITHUB_URL` The url for the product repository.
+Projects          | All of the product environment instance names should be added here.
+Rollbar           | The access token for the product's Rollbar account and your local Rollbar username go here.
+Slack             | Deployment and syncronisation scripts post to Slack to alert the team on various tasks. Settings for Slack are managed here.
```
Comment (KISS) [inline README.md]: syncronisation -> synchronization

**Review by KISS — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (devowhippit) [reply README.md]: Nyce catch

**Comment by devowhippit** [comment]
@KISS @kimpenguin @theasteve I added some additional updates and things I missed based on the walk through this morning. Usage section of readme is more simple with a notes section for the scary complicated stuff. Some bin scripts have been updated.

**Review by theasteve — APPROVED** [review]

**Review by kimpenguin — APPROVED** [review]
I super appreciate the `bin.cfg`! Digging the new scripts. The wpscan timed out on my, but I think its addition is super sweet. Great enhancements :tada:

**Comment by devowhippit** [comment]
@kimpenguin did you get a token from https://wpvulndb.com? It needs to be set for that script in the wpscan.yml file

**Comment by kimpenguin** [comment]
@devowhippit sure did! It seemed to be going well for a while and then it "exceeded the timeout of 300 seconds". :woman_shrugging: I'll revisit.

**Pull request merged**